src/models/user.ts

```diff
@@ -1,17 +1,32 @@
-import { v4 as uuidv4 } from 'uuid'
+import { UserInfoType } from './../types/userType'
+import { userUpdate } from '../types/userUpdate'
+import adapter from '../adapter'
 
-export default class User {
-  id: string
-  name: string
-  age: number
+export default class UserService {
+  userAdapter = new adapter('./routes/user', 'user.json')
```

Comment:
Hmmm, It wiil be better if yoo create instance of your adapter inside constructor method

Reply:
`userAdapter = new Adapter('./routes/user', 'user.json')`
https://google.github.io/styleguide/jsguide.html#naming

Reply:
userAdapter = new Adapter('./routes/user', 'user.json'). Everything good here, isn't it?
Need an advice how to add constructor like that

Reply:
https://google.github.io/styleguide/jsguide.html#naming